doc/endpoint/codecs.md

```diff
@@ -88,6 +88,89 @@ Different codec formats can be used in different contexts. When defining a path,
 with the `TextPlain` media type can be used. However, for bodies, any media type is allowed. For example, the 
 input/output described by `jsonBody[T]` requires a json codec.
 
+## Handling Delimited Path Parameters
+
+Tapir allows you to handle complex path parameters, such as lists of custom types separated by delimiters (e.g., commas). 
+This can be achieved using Codec.delimited, which facilitates the serialization and deserialization of delimited lists 
+within path segments.
+
+### Use Case
+Suppose you want to define an endpoint that accepts a list of names as a comma-separated path parameter. Each name should
+adhere to a specific pattern (e.g., only uppercase letters).
+
+### Implementation Steps:
+
+### 1. Define the Custom Type and Validator
+Start by defining your custom type and the associated validator to enforce the desired pattern.
+
+```scala
```

Comment:
this can probably be compilation-checked? you can do this by adding `mdoc:compile-only` and then running `compileDocumentation` from sbt to verify that everything works

Reply:
I ran that locally and got a success but needed to have all the code and imports together. I left `mdoc:compile-only` on the last code snippet but let me know if that should be removed.